eng/scripts/typespec/Test-Emitter.ps1

```diff
@@ -65,7 +65,8 @@ Push-Location $packageRoot
 try {
     Build-Emitter -packageRoot $packageRoot -testResultsPath $testResultsPath
 
-    if ($UnitTests) {
+    # we only run spector test for Azure emitter
+    if ($UnitTests -and $EmitterPackagePath.EndsWith("http-client-csharp")) {
```

Comment:
I hard-code this for now until we have separated the spector tests into arm and non-arm, we will run it for mgmt emitter